tokio-test/src/task.rs

```diff
@@ -0,0 +1,115 @@
+//! Futures task based helpers
```

Comment:
I know this module was copied from [`tokio-mock-task`](https://github.com/carllerche/tokio-mock-task), what's the motivation for reproducing the code rather than depending on that crate and re-exporting it?

Reply:
Not totally sure, but I feel that code is better to live here than it is in that repo.

Reply:
It would be fine to have it here TBH. No reason to have more crates... we can always add feature flags to this one.

I don't like publishing many crates...

Reply:
👍 In that case we should probably deprecate the `mock-task` crate? My concern was mostly around maintaining two separate copies of identical code.